scripts/docker-build.sh

```diff
@@ -39,7 +41,6 @@ cd ${basedir}
 echo "==== Installing Updates and Dependencies ===="
 export DEBIAN_FRONTEND=noninteractive
 echo "mysql setup..."
-mysql_install_db
```

Comment:
Why?

Reply:
Shifted to new script scripts/docker-prep_serv.sh with all other startup mysql code